server/app/views/components/TextFormatter.java

```diff
@@ -113,7 +112,8 @@ public static ImmutableList<DomContent> createLinksAndEscapeText(
       var urlTag =
           a().withText(url.getOriginalUrl())
               .withHref(url.getFullUrl())
-              .withClasses(BaseStyles.TEXT_SEATTLE_BLUE);
+              .withClasses(
+                  "text-blue-900", "font-bold", "opacity-75", "underline", "hover:opacity-100");
```

Comment:
This is here because of failures in the contrast ratio of the links to the page one links were added. Same classes as in the header/footer links.